src/actions/install.ts

```diff
@@ -0,0 +1,37 @@
+import ora from 'ora';
+import type { UsagePreference } from '../prompts/usage-preference';
+import { shell } from '../shell';
+import { Context, PackageManager } from './context';
+
+export function getInstallCommand(pkgManager: PackageManager): string {
+	return pkgManager === 'npm' ? 'install' : 'add';
```

Comment:
[`npm` had an `add` alias at least since version 9](https://docs.npmjs.com/cli/v9/commands/npm-install) which is the version which corresponds to the first node version that stylelint supports—i.e. 18.12.0.
The ternary is probably not necessary.

Reply:
Thanks for the clarification! And as far as I know it is similar in other package managers. Good catch!